docs/development/doc_guide.rst

```diff
@@ -1,76 +1,849 @@
+run:
+
+.. code-block:: bash
+
+   pip install -r requirements.txt
 
-Using sphinx within the project
+Building documentation with make
+--------------------------------
+
+If you have make_ installed, then you can build the documentation by
+entering the :file:`docs/` directory and running:
+
+.. code-block:: bash
+
+   make html
+
+Including the ``-j n`` flag in the ``make`` command will enable a
+parallel build, where ``n`` is replaced with the number of processes.
+
+You can access the documentation landing page by opening
+:file:`docs/_build/html/index.html` with your browser of choice.
+
+To remove all files previously generated by ``make``, run:
+
+.. code-block:: bash
+
+   make clean
+
+This command is needed when you make a change to a file that does not
+trigger Sphinx_ to rebuild the file that you altered, for example
+modifying a CSS_ file.
+
+Building documentation with tox
 -------------------------------
-To build docs locally, either:
 
-* use `Tox <https://tox.readthedocs.io/en/latest/>`_ with ``tox -e build_docs`` from within the main PlasmaPy repository directory, or
-* enter the ``docs`` directory and run ``make html``.
+You can use tox_ to locally build the documentation by running:
 
-Afterwards, open ``docs/_build/html/index.html`` with your browser of choice.
+.. code-block:: bash
 
-Do try to solve warnings in documentation when writing your code. To enforce this,
-The ``build_docs`` environment is set to fail on encountering any warnings via
-the ``-W`` flag to ``sphinx-build``
+   tox -e build_docs
 
-.. note::
-   The ``tox -e build_docs_no_examples`` command will build the documentation without
-   executing the :ref:`example notebooks <example_notebooks>`. It will also
-   pass with warnings.
+You can access the documentation landing page by opening
+:file:`docs/_build/html/index.html` with your browser of choice.
+
+When writing documentation, please make sure to fix any warnings that
+arise. To enforce this, the ``build_docs`` environment is set to fail on
+encountering any warnings via the ``-W`` flag to `sphinx-build`_.
+
+You can shorten the tox_ documentation build by running:
+
+.. code-block:: bash
+
+   tox -e build_docs_no_examples
+
+in order to build the documentation without executing the :ref:`example
+notebooks <example_notebooks>`. This command will pass even if there are
+warnings. A tox_ command may also include the ``--parallel`` flag to
+allow a parallel build.
+
+Documentation tools
+===================
+
+ReStructuredText
+----------------
+
+PlasmaPy's documentation is written using the `reStructuredText (reST)`_
+markup language. reST_ is human readable when viewed within a source
+code file or when printed out using `help`. reST_ also contains markup
+that allows the text to be transformed into `PlasmaPy's documentation`_.
+reST_ files end in :file:`.rst`. Documentation contained within
+:file:`.py` files are in the form of docstrings_, which are written in
+reST_.
```

Comment:
```suggestion
reST_ files use the file extension :file:`.rst`. Documentation contained
within :file:`.py` files are in the form of docstrings_, which are written in
reST_.
```

Reply:
I was wondering how to better word this.  Thanks!